extra-devel/avr-gcc/autobuild/beyond

```diff
@@ -0,0 +1,24 @@
+abinfo "Strip debug symbols from libraries"
+find "${PKGDIR}${_PREFIX}"/lib -type f -name "*.a" -exec /usr/bin/avr-strip --strip-debug {} \;
+
+abinfo "Install runtime library license exception"
+install -Dvm644 "${SRCDIR}/COPYING.RUNTIME" "${PKGDIR}/usr/share/licenses/avr-gcc/RUNTIME.LIBRARY.EXCEPTION"
+
+abinfo "Removing stray manuals"
+rm -vr "${PKGDIR}${_PREFIX}/share/man/man7"
+rm -vr "${PKGDIR}${_PREFIX}/share/info"
+
+abinfo "Removing locale"
+rm -vr "${PKGDIR}${_PREFIX}/share/locale"
+
+abinfo "Removing stray libcc1"
+rm -vr "${PKGDIR}${_PREFIX}/lib64"/libcc1.*
+rmdir -v "${PKGDIR}${_PREFIX}/lib64"
+
+abinfo "Creating symlinks in /usr/bin"
+mkdir -vp "${PKGDIR}/usr/bin"
+pushd "${PKGDIR}${_PREFIX}/bin"
+for i in avr-*; do
+	ln -sv "${_PREFIX}/bin/${i}" "${PKGDIR}/usr/bin/$i"
```

Comment:
Use relative paths for symlinks.

Reply:
This glob expression is protected by a pushd into an absolute path.